examples/free_tier_bucket/variables.tf

```
@@ -0,0 +1,16 @@
+variable "auth_token" {
+  description = "Authentication API Key"
+  sensitive   = true
+}
+
+variable "organization_id" {
+  description = "Capella Organization ID"
+}
+
+variable "project_id" {
+  description = "Project Name for Project Created via Terraform"
```

This is not the Project Name
```suggestion
  description = "Capella Project ID"
```

---

changed it here f5e0c0e1c6dfb991daf56098e56a8791db5de322